.github/workflows/ibmq_tests.yml

```diff
@@ -1,7 +1,7 @@
-name: IBMQ integration tests
+name: IBMQ integration tests with Qiskit 1.0
 on:
   schedule:
-    - cron: '0 0 * * 0,4' # At 00:00 on Sunday and Thursday.
+    - cron: '1 0 * * 0,4' # At 01:00 on Sunday and Thursday.

```

Comment:
why `01:00`? 🤔

Reply:
Oh actually I don't know. I deleted the old tests and renamed this one and I guess they had a different schedule? If it matters I can always change it back 😅